tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "ES5",
+		"target": "es5",
```

Comment:
I don't know which is correct ES5 or es5.
Complier options on TypeScript is seems to use ES5. I use "ES5". http://www.typescriptlang.org/docs/handbook/compiler-options.html